### ISSUE TITLE
Error pada jam 24.00 sd 06.00 + Senin sepanjang hari

### ISSUE THREAD

**Issue by g00m00nry0ng** [kadekjayak/bca-parser#18]
Ada yg sudah bisa fix error ketika jam jam 24.00 sd 06.00 dan setiap hari senin seperti hari ini? Di luar periode tsb, saya coba aman-aman saja, tapi pada periode tsb pasti error. Errornya yg muncul ketika cek saldo atau mutasi sbb: DOMDocument::loadHTML(): Document is empty in Entity, line: 2

error ini juga kejadian di repo sejenis, info terakhir sbb: https://github.com/ammarfaizi2/KlikBCA/issues/6#issuecomment-1364769670

**Comment by pancaz** [comment]
iya saya mengalami masalah yang sama hari ini senin tidak bisa cek saldo. mohon solusi ya.

**Comment by tarikhagustia** [comment]
kayaknya ini dari BCA nya deh, soalnya saya cek dari web nya langsung memang di hari senin tidak bisa cek mutasi

**Comment by g00m00nry0ng** [comment]
> kayaknya ini dari BCA nya deh, soalnya saya cek dari web nya langsung memang di hari senin tidak bisa cek mutasi

Nope. Sekarang di hari lain juga kejadian. Ga bisa cek mutasi dan saldo. Ga cuma repo ini sih. Semua repo juga ga bisa.

**Comment by tarikhagustia** [comment]
> > kayaknya ini dari BCA nya deh, soalnya saya cek dari web nya langsung memang di hari senin tidak bisa cek mutasi
> 
> Nope. Sekarang di hari lain juga kejadian. Ga bisa cek mutasi dan saldo. Ga cuma repo ini sih. Semua repo juga ga bisa.

Wah iya, harus pindah ke selenium kayaknya. ada library yang udah major kah?
